src/app/archive/archive_lib/test.ml

```diff
@@ -54,6 +54,12 @@ let%test_module "Archive node unit tests" =
       Mina_generators.User_command_generators.payment_with_random_participants
         ~keys ~max_amount:1000 ~fee_range:10 ()
 
+    let genesis_state_body =
```

Comment:
nit: this could be moved inside the definition of `genesis_state_view`, since it's not used anywhere else